libraries/AP_Math/SCurve.h

```diff
@@ -87,7 +87,7 @@ class SCurve {
     // target_pos should be set to this segment's origin and it will be updated to the current position target
     // target_vel and target_accel are updated with new targets
     // returns true if vehicle has passed the apex of the corner
-    bool advance_target_along_track(SCurve &prev_leg, SCurve &next_leg, float wp_radius, bool fast_waypoint, float dt, Vector3f &target_pos, Vector3f &target_vel, Vector3f &target_accel) WARN_IF_UNUSED;
+    bool advance_target_along_track(SCurve &prev_leg, SCurve &next_leg, float wp_radius, float accel_corner, bool fast_waypoint, float dt, Vector3f &target_pos, Vector3f &target_vel, Vector3f &target_accel) WARN_IF_UNUSED;
```

Comment:
It looks like we're missing descriptions of some of the arguments in the comments above this method.

- next_leg
- accel_corner